components/taxonomy/src/services.rs

```diff
@@ -158,6 +158,21 @@ pub enum ChannelKind {
     /// ```
     Ready,
 
+    /// Used when a setter takes no values.
+    ///
+    /// # JSON
+    ///
+    /// This kind is represented by string "NoKind".
+    ///
+    /// ```
+    /// use foxbox_taxonomy::services::*;
+    /// use foxbox_taxonomy::parse::*;
+    ///
+    /// let parsed = ChannelKind::from_str("\"NoKind\"").unwrap();
+    /// assert_eq!(parsed, ChannelKind::NoKind);
+    /// ```
+    NoKind,
```

Comment:
> That doesn't really make sense. A ChannelKind is supposed to describe a feature, not the type of the feature.

@Yoric, I thought channelKind was describing the data being sent over the channel? It feels redundant to have to create artificial channel kinds for each operation that has no data. It doesn't seem to add anything other than complexity.

Right now, it feels like I have to add 20-30 lines of taxonomy to call 1 line of adapter code.